helion/runtime/kernel.py

```diff
@@ -453,17 +453,24 @@ def autotune(
 
                 config = FiniteSearch(self, self.configs).autotune()
         else:
-            self.settings.check_autotuning_disabled()
-
-            from ..autotuner import DifferentialEvolutionSearch
-            from ..autotuner import LocalAutotuneCache
-
-            config = LocalAutotuneCache(
-                DifferentialEvolutionSearch(
-                    self,
-                    **kwargs,  # pyright: ignore[reportArgumentType]
-                ),
-            ).autotune()
+            if force or self.settings.use_default_autotuner:
+                self.settings.check_autotuning_disabled()
+
+                from ..autotuner import DifferentialEvolutionSearch
+                from ..autotuner import LocalAutotuneCache
+
+                config = LocalAutotuneCache(
+                    DifferentialEvolutionSearch(
+                        self,
+                        **kwargs,  # pyright: ignore[reportArgumentType]
+                    ),
+                ).autotune()
+            else:
+                config = self.config_spec.default_config()
+                log.warning(
+                    "Using default config for kernel: %s, please use @helion.autotune for better performance.",
```

Comment:
I'm worried people will forget this.  Running Helion kernels with default configs is a performance footgun.  

Any ideas on how we can make sure people don't deploy or benchmark default configs?  Maybe have a different kernel entrypoint for running with default configs...